Makefile

```diff
@@ -1,15 +1,50 @@
+# Variables
```

Comment:
@kgashok Can you review this file, as you have setup the Makefile.

Personally I don't we need to defined python and pip as global variables and the commands seems self explanatory so adding comments for each command looks like over-engineering.